force-app/main/default/classes/SOQL.cls

```diff
@@ -142,16 +200,17 @@ public with sharing class SOQL {
      * @return A list of Contact LastName.
      */
     public static List<String> getContactNamesByAccount(Id accountId) {
-        // Create a list to hold the Contact LastNames
+        List<String> contactLastNames = new List<String>();
 
-        // Query the Contacts
-        // Loop through each Contact and add the LastName to the list
-        //for () {
-        //    
-        //}
+        List<Contact> contacts = [SELECT Id, LastName, AccountId
+                                    FROM Contact
+                                    WHERE AccountId = :accountId];
 
-        // Return the list of Contact LastNames
-        return null; // Replace null with the variable you used to store the result
+            for (Contact contact : contacts) {
+                contactLastNames.add(contact.LastName);
+            }
```

Comment:
Good use case to try out the SOQL for loop. We don't care about the actual Contact records, just the Last Names.